### ISSUE TITLE
Will it work with open source models?

### ISSUE THREAD

**Issue by mariozupan** [leannchen86/raptor-rag-kg-enhanced#2]
Can it be used with some other llm, other then gpt, like llama, tiny lama and so on?

**Comment by leannchen86** [comment]
Of course. All you need to do is define your own language model pull in the API key (if any). Or, you can also use Ollama too, which allows you to access all open source models and run locally.

**Comment by mariozupan** [comment]
Great!!! I don't need API key with my fine-tuned model, I assume. We'll stay in contact.
